apt-sources/src/lib.rs

```diff
@@ -0,0 +1,425 @@
+#![deny(missing_docs)]
+//! A library for parsing and manipulating APT source files that
+//! use the DEB822 format to hold package repositories specifications.
+//!
+//! /// <div class="warning">
+//!
+//! Currently only lossy _serialization_ is implemented, lossless support
+//! retaining file sequence and comments would come at later date.
+//!
+//! </div>
+//! 
+//! # Examples
+//!
+//! ```rust
+//!
+//! use apt_sources::Repositories;
+//! use std::path::Path;
+//!
+//! let text = r#"Types: deb
+//! URIs: http://ports.ubuntu.com/
+//! Suites: noble
+//! Components: stable
+//! Architectures: arm64
+//! Signed-By:
+//!  -----BEGIN PGP PUBLIC KEY BLOCK-----
+//!  .
+//!  mDMEY865UxYJKwYBBAHaRw8BAQdAd7Z0srwuhlB6JKFkcf4HU4SSS/xcRfwEQWzr
+//!  crf6AEq0SURlYmlhbiBTdGFibGUgUmVsZWFzZSBLZXkgKDEyL2Jvb2t3b3JtKSA8
+//!  ZGViaWFuLXJlbGVhc2VAbGlzdHMuZGViaWFuLm9yZz6IlgQTFggAPhYhBE1k/sEZ
+//!  wgKQZ9bnkfjSWFuHg9SBBQJjzrlTAhsDBQkPCZwABQsJCAcCBhUKCQgLAgQWAgMB
+//!  Ah4BAheAAAoJEPjSWFuHg9SBSgwBAP9qpeO5z1s5m4D4z3TcqDo1wez6DNya27QW
+//!  WoG/4oBsAQCEN8Z00DXagPHbwrvsY2t9BCsT+PgnSn9biobwX7bDDg==
+//!  =5NZE
+//!  -----END PGP PUBLIC KEY BLOCK-----"#;
+//!
+//! let r = text.parse::<Repositories>().unwrap();
+//! let suites = r[0].suites();
+//! assert_eq!(suites[0], "noble");
+//! ```
+//!
+//! See the ``lossless`` module (behind the ``lossless`` feature) for a more forgiving parser that
```

Comment:
This comment seems copied from another crate, but there's no actual lossless module here?